PWGHF/TableProducer/trackIndexSkimCreator.cxx

```diff
@@ -1460,7 +1464,7 @@ struct HfTrackIndexSkimCreator {
   using FilteredHfTrackAssocSel = soa::Filtered<soa::Join<aod::HfTrackAssoc, aod::HfSelTrack>>;
   Preslice<FilteredHfTrackAssocSel> trackIndicesPerCollision = aod::hf_track_association::collisionId;
 
-  void process( // soa::Join<aod::Collisions, aod::CentV0Ms>::iterator const& collision, //FIXME add centrality when option for variations to the process function appears
+  void processSkimCreator( // soa::Join<aod::Collisions, aod::CentV0Ms>::iterator const& collision, //FIXME add centrality when option for variations to the process function appears
```

Comment:
It would be worth making the name more specific, (since there are now many different tasks in this workflow), something like `process2And3Prongs`.

Reply:
I renamed it following your suggestion as `process2And3Prongs` . For the moment I did not change the name of the dummy `processNoSkimCreator`. Is that OK or should I also rename it to `processNo2And3Prongs` to be consistent?